wren-ui/Dockerfile

```diff
@@ -50,12 +53,12 @@ COPY --from=builder /app/.next/standalone ./
 COPY --from=builder /app/.next/static ./.next/static
 COPY migrations ./migrations
 COPY knexfile.js ./knexfile.js
-RUN yarn global add knex
 
 EXPOSE 3000
 
 ENV PORT 3000
 
 # server.js is created by next build from the standalone output
 # https://nextjs.org/docs/pages/api-reference/next-config-js/output
-CMD knex migrate:latest && HOSTNAME="0.0.0.0" node server.js
+# use yarn dlx to run package without installing it
+CMD yarn dlx knex migrate:latest && HOSTNAME="0.0.0.0" node server.js
```

Comment:
`yarn dlx` will download package, hence will make server startup more unpredictable & time consuming. Could we still put `knex` installation in build steps?

Reply:
sth like this?
<img width="637" alt="image" src="https://github.com/user-attachments/assets/22e582f5-7e66-41c5-a383-b152d8d0ec3a" />

Reply:
`RUN yarn global add knex`

Reply:
fixed